R/filter_ard_hierarchical.R

```diff
@@ -125,6 +132,15 @@ filter_ard_hierarchical <- function(x, filter, keep_empty = FALSE) {
     )
   }
   ard_args <- attributes(x)$args
+  by <- ard_args$by
+
+  # get name of filtering variable
+  process_selectors(
+    as.list(ard_args$variables) |> data.frame() |> stats::setNames(ard_args$variables),
```

Comment:
Should this use `include` instead of `variables`, because they can only subset on include variables that have summary stats?

Reply:
I have a check just after this for `include` that I think returns a more informative message: https://github.com/insightsengineering/cards/pull/468/files#diff-f4499dac788ebee7d88eee5734b7aecfc4d8460aa7367a2b776cd5772aa8dbf6R152-R160

Does that look fine?